pages/[username]/vc/index.tsx

```diff
@@ -1,37 +1,39 @@
 import Layout from '../../../components/Layout'
-import { Headline, XPoints } from '../../../components'
+import { Headline, VerifiableCredentials } from '../../../components'
+import styles from '../../styles.module.scss'
 import { serverSideTranslations } from 'next-i18next/serverSideTranslations'
+import { GetStaticPaths } from 'next/types'
 
-function Component() {
+function VCIndex() {
     const data = [
         {
             name: 'DidDomains',
             router: '',
         },
     ]
-
     return (
         <>
             <Layout>
-                <div
-                    style={{
-                        width: '100%',
-                        marginTop: '10%',
-                        marginBottom: '-10%',
-                    }}
-                >
+                <div className={styles.headlineWrapper}>
                     <Headline data={data} />
                 </div>
-                <XPoints />
+                <VerifiableCredentials />
             </Layout>
         </>
     )
 }
 
+export const getStaticPaths: GetStaticPaths<{ slug: string }> = async () => {
```

Comment:
why do we have 2 getStaticPaths? @devwistkey

Reply:
Hi @tralkan we only have 1 getStaticPaths, the second one ":GetStaticPaths" is just only typescript declaration. Smae like :string :number etc